src/xform_to_datalog/datalog_facts.h

```diff
@@ -66,13 +67,17 @@ class DatalogFacts {
 .decl duplicateTestCaseNames(testAspectName: symbol)
 .output duplicateTestCaseNames(IO=stdout)
 
+.output disallowedUsage(IO=stdout)
```

Comment:
Why is the `.output disallowedUsage(IO=stdout)` declaration in two locations?

Reply:
See other comment.